doc1.md

```diff
@@ -0,0 +1,27 @@
+# Anna
+---
+### Arrangement of files
+
+* **css : This folder contains the plain text files format used for enhancing HTML files.**   
+  * Files named **_2.css_**,**_material.min.css_** ,**_on-off-switch.css_** and **_popup.css_** contains the layout for   popup.html.  
+  * **_cropper.css_** shows it's significance in **_background.js_** and is used to feature crop fuction.
+
+* **elate : This folder contains the files that are used to build the main Web-page of Anna(_index.html_).**
+  * **_index.html_** file is present under this folder which opens the main Web-page of Anna-Chrome Extension.
+  
+* **img : This folder contains _icon.png_ which potrays the icon of Anna.**
+
+* **js : This folder contains some of the main javascript files in order to support popup display,reversesearch,screenshot and crop feature.**
+  * Folder **_vendor_** along with files **_on-off-switch-onload.js_**,**_on-off-switch.js_**,**_on-off.js_**,**_popup.js_**,**_jquery-1.11.2.min.js_** designs the complete popup along with the on-off button effect.
+  * Folder **_cropperjs_** along with file **_content_script.js_** lays it's impression in adding Cropping feature to screenshot and reverse-serach.
+  * **_screenshot.js_** adds screenshot feature and is implemented in **_background.js_.**
+  
+ * **weatherFunctionality : This folder contains index.html which features Weather Function in the Extension.** 
```

Comment:
Omg this is still there😅 could you also send a pr to remove this folder as it's not needed